packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -85,33 +85,65 @@ const defaultProps = {
   panelSize: 'small',
 };
 
+/**
+ * This method ensures that aria-expanded attribute is allowed with the node role to avoid accessibility violations.
+ * More info on associated roles can be found here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles
+ * @param {HTMLElement} node - a node to check
+ * @returns true if node's role is compatible with aria-expanded attribute
+ */
+const roleIsCompartible = (node) => {
+  const role = node.getAttribute('role');
+  return (
+    role === 'application'
+    || role === 'button'
+    || role === 'checkbox'
+    || role === 'combobox'
+    || role === 'gridcell'
+    || role === 'link'
+    || role === 'listbox'
+    || role === 'menuitem'
+    || role === 'row'
+    || role === 'rowheader'
+    || role === 'tab'
+    || role === 'treeitem'
+    || role === null
+    || role === undefined
+  );
+};
```

Comment:
We should not need a method like this.  It makes me feel like we might be doing something incorrectly.

Reply:
Removed, the role in question changed in the example